src/frontend/web_application/src/components/MessageList/components/Message/index.jsx

```diff
@@ -100,7 +108,7 @@ class Message extends Component {
 
   render() {
     const { message, __ } = this.props;
-    const author = message.participants.find(participant => participant.type === 'from');
+    const author = message.participants.find(participant => participant.type === 'From');
```

Comment:
It's not a connected component. But yes we can eventually memoized the result using reselect. 
For now I think it is micro-optim. 
This kind of selector is almost everywhere, I will do an optimization check after this PR and may be apply memoization in that case if needed.